components/ReCaptcha.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+    <v-container v-if="((!this.captchaValidated) && (this.$config.NODE_ENV === 'production'))" fluid fill-height>
+        <v-row rows="auto" align-content="center" align="center">
+            <v-col cols="12" align-self="center" align="center">
+            <recaptcha class="align-center"/>
+            <v-btn @click="SubmitRecaptcha()" color="primary">Submit</v-btn> 
+            </v-col>
+        </v-row>
+    </v-container>
+    <v-container v-else-if="internalError">
```

Comment:
Maybe a single container tag and the v-if inside for the v-row or InternalError or UnderMaintenance